### DISCUSSION TITLE
why my github graph awlays shows dark green although i only make one or two contribution.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#135554] [New to GitHub]
![Screenshot 2024-08-12 230008](https://github.com/user-attachments/assets/0e29a3d6-f79b-4da3-90c4-49a8c8740b0a)

**Comment by 0xpuck** [comment]
GitHub dynamically scales the colors based on your activity. If your overall contributions are low, even one or two contributions might appear as dark green because it's a significant portion of your total activity.

And then, if you have consistent contributions over several days, the graph might reflect that by showing darker colors, even if each day's contributions are minimal.

**Reply by ghost** [reply]
> GitHub dynamically scales the colors based on your activity. If your overall contributions are low, even one or two contributions might appear as dark green because it's a significant portion of your total activity.
> 
> And then, if you have consistent contributions over several days, the graph might reflect that by showing darker colors, even if each day's contributions are minimal.

how to fix this issue ?

**Reply by 0xpuck** [reply]
What I mean is that you don't have to try to fix this.
The more you work on git, the more the colors will automatically become what you think they should be.
Don't worry, you can do everything in git, just get into the habit of doing it in git.

**Reply by ghost** [reply]
> What I mean is that you don't have to try to fix this. The more you work on git, the more the colors will automatically become what you think they should be. Don't worry, you can do everything in git, just get into the habit of doing it in git.

ok thanks ser!
